src/WebJobs.Script/Dispatch/JavaWorkerConfigProvider.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Abstractions.Rpc;
+using Microsoft.Azure.WebJobs.Script.Config;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.Azure.WebJobs.Script.Dispatch
+{
+    public class JavaWorkerConfigProvider : IWorkerConfigProvider
+    {
+        public WorkerDescription GetDescription() => new WorkerDescription
+        {
+            Language = "Java",
+            Extension = ".jar",
+            DefaultWorkerPath = "azure-functions-java-worker.jar",
+        };
+
+        public bool TryConfigureArguments(ArgumentsDescription args, IConfiguration javaSection, ILogger logger)
+        {
+            var settings = ScriptSettingsManager.Instance;
```

Comment:
Can we avoid depending on the static instance (avoid depending on `ScriptSettingsManager` altogether)? 
If the worker needs access to the root configuration it should require it in its constructor.
The goal is to get rid of `ScriptSettingsManager` in the runtime, so if we can avoid additional use, that would help.